next.config.mjs

```diff
@@ -25,7 +25,16 @@ const nextConfig = {
   },
   experimental: {
     scrollRestoration: true
-  }
+  },
+  redirects: () => {
+    return [
+      {
```

Comment:
Next redirects aren't supported for static builds. Need to use the `_redirects` file. This is where CF Pages is failing.